extensions/amp-live-list/0.1/test/test-amp-live-list.js

```diff
@@ -255,6 +255,22 @@ describe('amp-live-list', () => {
     }).to.throw(/must have an "items" slot/);
   });
 
+  it('should have aria-live=polite by default', () => {
```

Comment:
actual code is testing the override case though (same issue with test-amp-fresh.js as well)  Let's have two tests, one for default and one for override.